### PR TITLE
Fix issue 197

### PR DESCRIPTION
**Pull request by ammoheim**
*Issue #, if available:*
#197
*Description of changes:*
added sam quickstart tempate for SQS for dotnet6

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by ulili5 — APPROVED** [review]

**Review by ulili5 — APPROVED** [review]

File: tests/integration/unit_test/test_unit_test_dotnet6.py [hunk 1]
```diff
@@ -18,4 +18,7 @@ class UnitTest_dotnet6_cookiecutter_aws_sam_hello_step_functions_sample_app(Unit
     code_directories = ["tests/StockBuyer.Test", "tests/StockChecker.Test", "tests/StockSeller.Test"]
 
 
+class UnitTest_dotnet6_cookiecutter_aws_sam_quick_start_sqs(UnitTestBase.DotNetCoreUnitTestBase):
+    directory = "dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet"
+    code_directories = ["test/FunctionTest.cs"]
```
Comment (ulili5) [inline tests/integration/unit_test/test_unit_test_dotnet6.py:23]: @ammoheim This step is making the test to fail. 
Let's fix it

File: dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/test/{{cookiecutter.project_name}}.Tests/FunctionTest.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+using Xunit;
```
Comment (ulili5) [inline dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/test/{{cookiecutter.project_name}}.Tests/FunctionTest.cs:1]: Alter the Teste project folder name 
from:
{{cookiecutter.project_name}}/test/{{cookiecutter.project_name}}.Tests/FunctionTest.cs
to:
{{cookiecutter.project_name}}/test/project.Tests/FunctionTest.cs

File: dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/test/{{cookiecutter.project_name}}.Tests/{{cookiecutter.project_name}}.Tests.csproj [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (ulili5) [inline dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/test/{{cookiecutter.project_name}}.Tests/{{cookiecutter.project_name}}.Tests.csproj:1]: Alter the Teste project folder name 
from:
{{cookiecutter.project_name}}/test/{{cookiecutter.project_name}}.Tests/FunctionTest.cs
to:
{{cookiecutter.project_name}}/test/project.Tests/FunctionTest.cs

File: tests/integration/unit_test/test_unit_test_dotnet6.py (hunk 1 above)
Comment (ulili5) [reply tests/integration/unit_test/test_unit_test_dotnet6.py:23]: Point to test/project.Tests after renaming the folder name

**Review by ulili5 — CHANGES_REQUESTED** [review]
Alter the test Folder project to project.Test and point to this folder name from the python integration test

File: dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/test/{{cookiecutter.project_name}}.Tests/FunctionTest.cs (hunk 2 above)
Comment (ammoheim) [reply dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/test/{{cookiecutter.project_name}}.Tests/FunctionTest.cs:1]: fixed

File: tests/integration/unit_test/test_unit_test_dotnet6.py [hunk 4]
```diff
@@ -18,4 +18,7 @@ class UnitTest_dotnet6_cookiecutter_aws_sam_hello_step_functions_sample_app(Unit
     code_directories = ["tests/StockBuyer.Test", "tests/StockChecker.Test", "tests/StockSeller.Test"]
 
 
+class UnitTest_dotnet6_cookiecutter_aws_sam_quick_start_sqs(UnitTestBase.DotNetCoreUnitTestBase):
+    directory = "dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet"
+    code_directories = ["test/project.Tests.cs"]
```
Comment (ulili5) [inline tests/integration/unit_test/test_unit_test_dotnet6.py:23]: @ammoheim can you please correct this line of code to remove .cs?
from:
["test/project.Tests.cs"]
to:
["test/project.Tests"]

File: dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet/cookiecutter.json [hunk 5]
```diff
@@ -0,0 +1,4 @@
+{
+    "project_name": "Name of the project",
+    "runtime": "dotnet6"
```
Comment (ulili5) [inline dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet/cookiecutter.json:3]: @ammoheim can you please add these proprierties to make it easy for customer to select Graviton 2 (ARM64) architecture?

[https://github.com/aws/aws-sam-cli-app-templates/blob/6aaf6ac2e0ada5f87c4c6f69c5c1d8696513f2fe/dotnet6/cookiecutter-aws-sam-hello-step-functions-sample-app/cookiecutter.json#L4](https://github.com/aws/aws-sam-cli-app-templates/blob/6aaf6ac2e0ada5f87c4c6f69c5c1d8696513f2fe/dotnet6/cookiecutter-aws-sam-hello-step-functions-sample-app/cookiecutter.json#L4)

File: dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/template.yaml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-sqs-queues.html
+  SimpleQueue:
+    Type: AWS::SQS::Queue
+  
+  # This is the Lambda function definition associated with the source code: sqs-payload-logger.js. For all available properties, see
+  # https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+  SQSPayloadLogger:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: ./src/{{cookiecutter.project_name}}/
```
Comment (ulili5) [inline dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/template.yaml:33]: @ammoheim and also here to add Graviton support 

[https://github.com/aws/aws-sam-cli-app-templates/blob/6aaf6ac2e0ada5f87c4c6f69c5c1d8696513f2fe/dotnet6/cookiecutter-aws-sam-hello-step-functions-sample-app/%7B%7Bcookiecutter.project_name%7D%7D/template.yaml#L55](https://github.com/aws/aws-sam-cli-app-templates/blob/6aaf6ac2e0ada5f87c4c6f69c5c1d8696513f2fe/dotnet6/cookiecutter-aws-sam-hello-step-functions-sample-app/%7B%7Bcookiecutter.project_name%7D%7D/template.yaml#L55)

**Review by ulili5 — CHANGES_REQUESTED** [review]
@ammoheim 2 minor changes on the comments

**Review by ulili5 — APPROVED** [review]
Looks good for me...

**Review by jfuss — APPROVED** [review]

**Pull request merged**